TheWarWithin/WarriorProtection.lua

```diff
@@ -1892,6 +1903,15 @@ spec:RegisterAbilities( {
     },
 } )
 
+local NewFeature = "|TInterface\\OptionsFrame\\UI-OptionsFrame-NewFeatureIcon:0|t"
+
+spec:RegisterSetting( "spell_reflection_filter", true, {
```

Comment:
You can add this option to Fury as well; it will inherit Spell Reflection from Arms since Arms is loaded first.

Reply:
Will do!